plugins/ssr/ssrcommon.ml

```diff
@@ -1223,7 +1223,7 @@ let abs_wgen keep_let f gen (gl,args,c) =
   let evar_closed t p =
     if occur_existential sigma t then
       CErrors.user_err ?loc:(loc_of_cpattern p) ~hdr:"ssreflect"
-        (pr_constr_pat (EConstr.Unsafe.to_constr t) ++
+        (pr_constr_pat env sigma (EConstr.Unsafe.to_constr t) ++
```

Comment:
Should `pr_constr_pat` be switched to `EConstr`?

Reply:
Yes, but that's orthogonal to this PR.